alembic/versions/ed775af5e086_rdiscrowd_4967_add_partial_index_for_.py

```diff
@@ -0,0 +1,27 @@
+"""RDISCROWD-4967 add partial index for task table
+
+Revision ID: ed775af5e086
+Revises: 29c0cc545d6d
+Create Date: 2022-02-28 15:01:53.582867
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'ed775af5e086'
+down_revision = '29c0cc545d6d'
+
+from alembic import op
+
+
+def upgrade():
+    op.create_index('task_state_calibration_exported_idx', 'task',
+                    ['id', 'created', 'project_id', 'state', 'quorum',
+                     'calibration', 'priority_0', 'info', 'n_answers',
+                     'fav_user_ids', 'exported', 'user_pref', 'worker_pref',
+                     'worker_filter', 'gold_answers', 'expiration'],
```

Comment:
Do we need to pass full column list when table name is passed as second param?

Reply:
Thanks for the explanation. This is a useful info and can be documented for future reference as 
```python
# passing columns list with index will avoid DB trip to query original table making indexing efficient.
columns = ["id", "created", ...]
op.create_index('task_state_calibration_exported_idx', 'task', columns, ...)
```